.eslintrc

```diff
@@ -45,5 +45,8 @@
   },
   "plugins": [
     "react"
-  ]
+  ],
+  "globals": {
+        OPIDC_HOST: true
```

Comment:
We should use OIDC as opposed to OPIDC. I have never seen OPIDC used anywhere before this.